<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Interface</class>
 <widget class="QDialog" name="Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>361</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1</width>
    <height>1</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>10000</width>
    <height>10000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Serial Port Configuration</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>WStreamLab.ico</normaloff>WStreamLab.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton:hover {
       background-color: #d7d9d7;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_1">
         <property name="styleSheet">
          <string notr="true">QGroupBox  {
    border: 2px solid gray;
    border-color: #FF17365D;
    background-color: #E6E6FF;
	font-family: &quot;Segoe UI&quot;;
	font-size: 12pt;
    margin-top: 10px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 7px;
    padding: -5px 5px 0px 5px;
}

QCheckBox::indicator {
    width: 15px;
    height: 15px;
    background: rgb(100, 100, 100);
 }

QCheckBox::indicator:checked
 {
    width: 15px;
    height: 15px;
    background: rgb(240, 255, 240);
 }

QCheckBox {
	font-family: &quot;Segoe UI&quot;;
	font-size: 12pt;
}

QSpinBox {
	background: rgb(240, 255, 240);
	font-family: &quot;Segoe UI&quot;;
	font-size: 12pt;
}

QLineEdit {	
	background: rgb(240, 255, 240);
	selection-background-color: rgb(233, 99, 0);
	font-family: &quot;Segoe UI&quot;;
	font-size: 12pt;
 }</string>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout">
            <item row="4" column="0">
             <widget class="QLabel" name="lbSelectStopBits_1">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="lbTimeout_1">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="lbSelectSerial_1">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="cbSelectDataBits_1">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="lbDataBits_1">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="cbBaudRate_1">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QComboBox" name="cbSelectStopBits_1">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="cbSelectSerial_1">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="lbSelectParity_1">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="leTimeout">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="lbBaudRate_1">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QComboBox" name="cbSelectParity_1">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="lbNumberOfRetries_1">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QComboBox" name="cbNumberRetries_1">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="styleSheet">
          <string notr="true">QGroupBox  {
    border: 2px solid gray;
    border-color: #FF17365D;
    background-color: #E6E6FF;
	font-family: &quot;Segoe UI&quot;;
	font-size: 12pt;
    margin-top: 10px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 7px;
    padding: -5px 5px 0px 5px;
}

QCheckBox::indicator {
    width: 15px;
    height: 15px;
    background: rgb(100, 100, 100);
 }

QCheckBox::indicator:checked
 {
    width: 15px;
    height: 15px;
    background: rgb(240, 255, 240);
 }

QCheckBox {
	font-family: &quot;Segoe UI&quot;;
	font-size: 12pt;
}

QSpinBox {
	background: rgb(240, 255, 240);
	font-family: &quot;Segoe UI&quot;;
	font-size: 12pt;
}

QLineEdit {	
	background: rgb(240, 255, 240);
	selection-background-color: rgb(233, 99, 0);
	font-family: &quot;Segoe UI&quot;;
	font-size: 12pt;
 }</string>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="4" column="0">
             <widget class="QLabel" name="lbSelectStopBits_2">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="lbTimeout_2">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="lbSelectSerial_2">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>150</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="cbSelectDataBits_2">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="lbDataBits_2">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="cbBaudRate_2">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QComboBox" name="cbSelectStopBits_2">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="cbSelectSerial_2">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="lbSelectParity_2">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="leTimeout_2">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="lbBaudRate_2">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>150</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QComboBox" name="cbSelectParity_2">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="lbNumberOfRetries_2">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QComboBox" name="cbNumberRetries_2">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>259</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pbRefreshSerialPort">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true"> padding-left: 10px; 
 padding-top: 1px; 
 padding-right: 10px;
 padding-down: 10px; </string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbTestConnection">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true"> padding-left: 10px; 
 padding-top: 1px; 
 padding-right: 10px;
 padding-down: 10px; </string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbSaveConfiguration">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true"> padding-left: 10px; 
 padding-top: 1px; 
 padding-right: 10px;
 padding-down: 10px; </string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbClose">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true"> padding-left: 10px; 
 padding-top: 1px; 
 padding-right: 10px;
 padding-down: 10px; </string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbSelectSerial_1</tabstop>
  <tabstop>cbBaudRate_1</tabstop>
  <tabstop>cbSelectDataBits_1</tabstop>
  <tabstop>cbSelectParity_1</tabstop>
  <tabstop>cbSelectStopBits_1</tabstop>
  <tabstop>leTimeout</tabstop>
  <tabstop>cbNumberRetries_1</tabstop>
  <tabstop>pbRefreshSerialPort</tabstop>
  <tabstop>pbTestConnection</tabstop>
  <tabstop>pbSaveConfiguration</tabstop>
  <tabstop>pbClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
